content/inside-rust/program-management-update-2025-08.md

```diff
@@ -0,0 +1,270 @@
+
+Every six month, half of the Council's term ends.
+
+Infra, Lang, Libs and Mods teams are looking for a new representative and the nominations are now open.
+
+If you want to learn more or you're interested in representing your team, please read [Eric Huss's post announcing the selection](https://blog.rust-lang.org/inside-rust/2025/08/15/leadership-council-repr-selection/).
+
+
+## Rust Foundation Project Directors 2025
+
+This fall, [we're also looking for new Project Directors][pd-blog].
+
+[pd-blog]: https://blog.rust-lang.org/inside-rust/2025/08/20/electing-new-project-directors-2025/
+
+The Directors have staggered terms as well and half is up for election every year.
+
+This time it's Santiago Pastorino, Scott McMurray, and Jakob Degen's. None are seeking reelection.
+
+These are seats directly on the Rust Foundation board. The Project directors serve the interest of the Rust Project as a whole and sit alongside the Member Directors who represent companies funding the Foundation.
+
+Each resolution the Foundation passes must be approved by *both* Member and Project Directors separately. That means regardless of the size of the board, the Project has an equal voice in Foundation matters.
+
+You can nominate yourself or another person until 2025-09-18. Please [read the blog post for more information][pd-blog]. The Project is always looking for fresh faces and diverse voices.
+
+I am the [facilitator of the selection process](https://github.com/rust-lang/leadership-council/blob/main/policies/project-directorship/election-process.md#setup) this time around. That means I've authored the blog post above, proposed the timeline, and I'll seek out consent and statements from the nominees. I've also announced the election on zulip as well as an email that should reach all Project members and I'll see it all through, including facilitating the actual election process.
+
+## Bevy/gamedev followup
+
+A month ago, the Lang team invited the [Bevy game engine folks](https://bevy.org/) to talk about issues faced by their new users as well as any pain points the project is facing.
+
+There were two big topics they mentioned: reflection and variadic generics.
+
+Both have been requested for a long time and the interest is much broader than just Bevy or even just the game development space.
+
+### Reflection
+
+Reflection is a mechanism that lets your program look at any type and understand it: getting its name, fields and *their names and types* while your program is running. This is in contrast to the `derive` macro or trait bounds that are processed at compile time.
+
+Projects like Bevy currently rely on the `derive` macros. For example, any component in its [ECS (Entity Component System)](https://bevy.org/learn/quick-start/getting-started/ecs/) must be annotated with `#[derive(Component)]`. While the usage is simple, these macros are difficult to write and debug. And the language has limitations on where they can be applied.
```

Comment:
Seconding Promethia's critique of this section. `Component` impls are extremely simple, and we could comfortably move to a macro-free design (either via blanket or manual impls) if we had to. Instead, the code generation occurs inside of the `Reflect` derive, from `bevy_reflect`. This derive is optional, but allows people to do interesting things for tooling and serialization, so it's present on virtually all of our data types.

`bevy_reflect` is also fully decoupled from the rest of Bevy as an ECS or game engine. We even have a few users *just* using it :)

The sections below on the pain of the orphan rule all still basically ring true if you replace `Component` with `Reflect`.